tests/unit/amo/pages/TestUserProfileEdit.js

```diff
@@ -636,23 +618,6 @@ describe(__filename, () => {
     );
   });
 
-  it('disables the submit button when username is empty', () => {
-    const root = renderUserProfileEdit();
```

Comment:
ah right, maybe we can reuse the logic for `username` but applied to `displayName`.

Reply:
> adjust this "optional" text?

Heh. I overlooked this part of your comment, but we definitely have to do something about that. I don't think it is a blocker, but the PR is not that big, so that could likely fit in this PR.

Reply:
I agree.. it shouldn't be a blocker if you'd rather address this later

Reply:
I re-added the logic to prevent the submit button to be clicked. I filed #7301 for the UI.